README.md

```diff
@@ -34,6 +34,12 @@ Install requirements
 pip install -r requirements.txt
 ```
 
+# Linux Additionals
```

Comment:
chdman comes in the mame-tools package? That's pretty cool if that's the case. Can you please say in the description why this package is needed? Just adding a little " ...to get mame-tools (for chdman)" would be fine.

Reply:
done! (hope github merge/update the pull request with my changes)